### DISCUSSION TITLE
Is Split intended to make "smart clips" contain all the audio from the parent clip

### DISCUSSION THREAD

**Discussion by petersampsonaudacity** [audacity/audacity#4137] [Ideas & Feature Requests]
Are the  Split commands (Edit > Audio Clips > Split / Split New) intended, i.e. by intentional design, to make "smart clips" such that each clip after the split contains all the audio of the originating clip (albeit with some audio hidden)?

I ask this as this is not documented in either the GitBook or the Manual:
https://support.audacityteam.org/basics/audacity-editing
https://alphamanual.audacityteam.org/man/Edit_Menu:_Clip_Boundaries

And it was behavior that totally surprised _(and indeed confused)_ me when I was testing this bug that @billw58  posted yesterday:
 **Sometimes a pasted smart clip can't be expanded to the right #4134** 

Since it caught me by surprise it may well catch out other unsuspecting users too - so if it is this way "by design" then it really ought to be documented.

This is made worse by the fact that the creation of the "smart clips" like this is invisible to the user as we still have no visual cue to indicate the presence of hidden data in a "smart clip"
 **There is no visual cue for a shrunk clip #1912** 

Tagging @Tantacrul @DilsonsPickles and @LWinterberg  as a design/UX matter

**Comment by billw58** [comment]
All audio exists as clips. All clips are smart. These are first things a user needs to know.

This note on the audio-editing page is ambiguous.
> Note: Trimming is a non-destructive operation. You can un-trim a clip at any time. If you have created your clip by splitting up a larger clip, you can even un-trim the current clip until it's at the **length of the old clip**.  


It depends on what you mean by "length". 

If the "larger clip" is a sub-clip of an even larger clip, then the "current clip" can be expanded to encompass the audio contained in the "even larger clip". This can go up the hierarchy until the "current clip" **may** contain all the audio in the track (assuming the track is all one clip). So "length" means "all the audio contained in the original clip, including any hidden audio", **not** "the length (in seconds) of the original clip".

This is difficult to document clearly and concisely. In the context of that editing tutorial I think what's there is about as deep as you want to go.

The full description of how clips work seems to me to be "reference" material, which (as I understand it) Muse is not keen to maintain. Perhaps a tutorial on clips would be a clearer way to get the nuances across.

One more thing ...
One can create clips without using the "special" clip-making commands. Copying or cutting a selection and pasting it into a new track creates a (smart) clip. Selecting a region and Duplicating it creates a clip.

Just remember: all clips are smart, and they contain all the audio (including hidden audio) of the clip they came from.

**Reply by petersampsonaudacity** [reply]
> The full description of how clips work seems to me to be "reference" material, which (as I understand it) Muse is not keen to maintain. >Perhaps a tutorial on clips would be a clearer way to get the nuances across.

I agree with your second part, but not so much you first pat @billw58 

The use of and management of clips seems to me to be a somewhat complex issue that needs and deserves a tutorial or how-to approach.

Muse's intention (my understanding anyway) is to move any/all tutorial  and how-to type material to the Gitbook - some has already migrated - leaving behind a rump "Reference Manual" (which currently I am maintaining, and you too if you wish to continue Bill).  I have prepped for this scenario by restructuring the front page of the 3.2.3 and alpha  manuals by splitting how-tos and tutorials off into sub-pages of the main landing page and renamed the front page as "... Reference Manual":
https://manual.audacityteam.org/

Yes there is some basic reference material about clips which would need to go in the Reference Manual (for instance the fact that Split produces smart clips etc, - but I would expect to be referencing a full-blown tutorial on Clips usage in the Gitbook, and such a tutorial does not yet exist.

> One more thing ... One can create clips without using the "special" clip-making commands. 

Indeedy, just recording or importing a track will get you a clip which is a "smart clip"

**Reply by petersampsonaudacity** [reply]
> Yes there is some basic reference material about clips which would need to go in the Reference Manual (for instance the fact that Split produces smart clips etc, - but I would expect to be referencing a full-blown tutorial on Clips usage in the Gitbook, and such a tutorial does not yet exist.

OK so I have done the prep work by adding a raft of P1s to the Manual for "smart clips" info (some my be downgradable to P2, I can look at that later).

The really important ones are (I think):
a) https://alphamanual.audacityteam.org/man/Audacity_Tracks_and_Clips
b) https://alphamanual.audacityteam.org/man/Edit_Menu:_Clip_Boundaries
c) https://alphamanual.audacityteam.org/man/Edit_Menu:_Remove_Special

**Comment by petersampsonaudacity** [comment]
This is pretty much all that the GitBook has t say about "smart clips" _(currently at least)_:
![image](https://user-images.githubusercontent.com/50205984/210600914-39e5911d-342d-4f01-9982-64e1ee548055.png)

It's not entirely surprising that we are getting many reports on the Forum of users encountering problems and issues with them.

**Comment by petersampsonaudacity** [comment]
And this section gives no indication that each new clip will be a "smart clip" that can be expanded to the full data set of the originating clip:
![image](https://user-images.githubusercontent.com/50205984/210602664-f000ca2b-fd5a-4302-b7ce-4dafaa346602.png)

**Comment by LWinterberg** [comment]
The trimming/untrimming will be made more discoverable in a future release. I believe you've seen these already.
![image](https://user-images.githubusercontent.com/87814144/210603182-5f5b7ebc-9d77-4535-8c34-5e9ac0a9d5eb.png)

**Reply by petersampsonaudacity** [reply]
> ...I believe you've seen these already. 

Ooh, I don't think so

**Comment by petersampsonaudacity** [comment]
I think our work on the Reference Manual for smart clips is now done. now that @billw58 and I have worked on the various relevant pages,
